plugins/task-plugin/package.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
package-lock.json should be removed (we're using yarn workspaces)

Reply:
removed, thank you!

Reply:
removed, thank you!